packaging/pyinstaller/libertem-server.spec

```diff
@@ -0,0 +1,68 @@
+# -*- mode: python ; coding: utf-8 -*-
+
+import sys ; sys.setrecursionlimit(sys.getrecursionlimit() * 5)
+
+dask_datas = []
+
+dask_files = [
+    ('Lib/site-packages/dask/dask.yaml', 'dask'),
+    ('Lib/site-packages/distributed/distributed.yaml', 'distributed')
```

Comment:
This could be handled in more portable way by creating a hook for `distributed` that calls `collect_data_files('distributed')`.